cluster/images/etcd/.gitignore

```diff
@@ -0,0 +1,4 @@
+etcd
+etcdctl
+etcd-*-linux-amd64.tar.gz
+etcd-*-linux-amd64/
```

Comment:
I think @wojtek-t was referring to the weird circle with slash that appears in GitHub's diff view. Hovering over indicates this means there's no newline at the end of this file, so maybe add one?